template/lambda-sqs-worker-cdk/.buildkite/pipeline.yml

```diff
@@ -56,6 +61,11 @@ steps:
       - *docker-ecr-cache
       - docker-compose#v4.16.0:
           run: app
+          environment:
+            - ENVIRONMENT
+            - GITHUB_API_TOKEN
+            - VERSION
```

Comment:
Any ideas where `VERSION` comes from? @72636c. I can see `ENVIRONMENT` in the UI so it will get propagated successfully by the `propagate-environment` setting.

Reply:
https://github.com/ajain58/skuba/blob/bbdd1782c27d400ec0abc4908246ee206fd4d6dd/template/lambda-sqs-worker/.buildkite/pipeline.yml#L47-L48

Is this template meant to have this line?

Reply:
Ah yes, I think that's right re: `VERSION`. Maybe we should extend the CDK template to use the version variable too

CDK

https://github.com/seek-oss/skuba/blob/454710c4ab068c073918ec2426896756169ce22d/template/lambda-sqs-worker-cdk/src/app.ts#L4-L6

Serverless

https://github.com/seek-oss/skuba/blob/454710c4ab068c073918ec2426896756169ce22d/template/lambda-sqs-worker/src/config.ts#L48

https://github.com/seek-oss/skuba/blob/454710c4ab068c073918ec2426896756169ce22d/template/lambda-sqs-worker/src/framework/logging.ts#L14-L17